### PR TITLE
Remove status, create plot census and forest census

### PR DESCRIPTION
**Pull request by mirunaish**
removed status table, added plot census table, forest census table, plot census assignment through table, plot census column in tree census

### CODE REVIEW

File: packages/schema/src/forest-census.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+   */
+  id: string;
+
+  /**
+   * Name assigned to this census
+   */
+  name: string;
+
+  /**
+   * Whether this census is still in progress or finished
```
Comment (julian-george) [inline packages/schema/src/forest-census.ts:19]: Maybe make all schema fields optional?

File: packages/server/src/db/migrations/20220415031244-create-forest-plot-census-tables.js [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+      const forests = await queryInterface.sequelize.query(
+        'SELECT forests.id FROM tree_census LEFT OUTER JOIN trees ON tree_census."treeTag" = trees.tag LEFT OUTER JOIN plots ON trees."plotNumber" = plots.number LEFT OUTER JOIN forests ON plots."forestId" = forests.id;',
+        {
+          transaction,
+          type: Sequelize.QueryTypes.SELECT,
+        }
+      );
+
+      // remove duplicates
+      var uniqueForestIds = [];
```
Comment (julian-george) [inline packages/server/src/db/migrations/20220415031244-create-forest-plot-census-tables.js:129]: nit but replace `var` with `const`, you can push/edit to arrays set with `const` as long as you dont give them a new pointer

File: packages/server/src/db/migrations/20220415031244-create-forest-plot-census-tables.js [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+      const plots = await queryInterface.sequelize.query(
+        'SELECT plots.number AS "plotNumber", forest_census.id AS "forestCensusId" FROM tree_census LEFT OUTER JOIN trees ON tree_census."treeTag" = trees.tag LEFT OUTER JOIN plots ON trees."plotNumber" = plots.number LEFT OUTER JOIN forests ON plots."forestId" = forests.id LEFT OUTER JOIN forest_census ON forests.id = forest_census."forestId";',
+        {
+          transaction,
+          type: Sequelize.QueryTypes.SELECT,
+        }
+      );
+
+      // remove duplicates
+      var uniquePlots = [];
```
Comment (julian-george) [inline packages/server/src/db/migrations/20220415031244-create-forest-plot-census-tables.js:163]: same nit here

**Comment by julian-george** [comment]
make sure to remove the tree_status stuff from the seeders file. e.g. remove the `bulkInsert` into the now-deleted `tree-statuses` table, and remove all of the `statusName` stuff and other foreign keys to tree statuses.

**Review by ziruihao — APPROVED** [review]
Overall awesome work! Definitely a lot of new tables but this is some really good, organized code! Thanks for making non-columns optional!

**Pull request merged**